packages/config/test/index.ts

```diff
@@ -462,14 +433,12 @@ test('normalizing the value of public-hoist-pattern', async (t) => {
       },
     })
 
-    t.equal(config.publicHoistPattern, undefined)
+    expect(config.publicHoistPattern).toBeUndefined()
   }
-  t.end()
 })
 
-test('rawLocalConfig in a workspace', async (t) => {
+test('rawLocalConfig in a workspace', async () => {
```

Comment:
This test fails locally if you have a `.npmrc` global file, the outcome does not match

Reply:
Actually also fails here, maybe I did something wrong

Reply:
Let's skip the failing test. We'll need to refactor the config reader.

Reply:
Ok, I'll add a comment and skip those.